### PR TITLE
Code Quality: Clean up shareable components between checklist and help center

### LINKED ISSUE #8122: Clean up shareable components between checklist and help center
## Feature Description

In release 1.9 we moved the prepublish checklist out of the inspector panel group and into a popup next to the help center to toggle open. Now that everything is in its proper place for that added feature, we want to go back and clean up duplicated components surrounding the popup functionality to dry up code. 

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

Storybook for this would be good to add, any relevant tests
Remove duplicate code, make sure that directory structures and naming make sense.

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Part of cleanup after ppcv2

## Summary
Just cleans up some of the dependencies between the ppcv2 popup and the help center popup so the ppcv2 component isn't dependent on the help center

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
NA

## Testing Instructions
Open and play with the help center, checklist & shortcuts menu in the bottom right corner of the editor and see that no behavior has changed.

### QA

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #8122

### CODE REVIEW

File: assets/src/edit-story/components/helpCenter/navigator/index.js [hunk 1]
```diff
@@ -19,48 +19,21 @@
...
 import { POPUP_ID } from '../constants';
-import {
-  DISTANCE_FROM_TOP,
-  DISTANCE_FROM_BOTTOM,
-} from '../../checklist/styles';
+import { NavigationWrapper, TopNavigation } from '../../secondaryPopup';
 import { BottomNavigation } from './bottomNavigation';
-import { NAVIGATION_WIDTH } from './constants';
-import { TopNavigation } from './topNavigation';
+// import { TopNavigation } from './topNavigation';
```
Comment (samwhale) [inline assets/src/edit-story/components/helpCenter/navigator/index.js]: ~👍 ~ I totally read this wrong. I thought this was being un-commented 🤕. Thanks Pascal for finding that

File: assets/src/edit-story/components/transition/scheduledTransition.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const machine = {
     entering: { entered },
     entered: { exiting },
     exiting: { exited },
-    exited: {},
+    exited: { entering },
```
Comment (samwhale) [inline assets/src/edit-story/components/transition/scheduledTransition.js:35]: 👍

**Review by samwhale — APPROVED** [review]
Rip to having two `ScheduledTransition`s

File: assets/src/edit-story/components/helpCenter/navigator/index.js (hunk 1 above)
Comment (swissspidy) [inline assets/src/edit-story/components/helpCenter/navigator/index.js]: Just remove instead of commenting out?

Comment (maxyinger) [reply assets/src/edit-story/components/helpCenter/navigator/index.js]: ope. yup. thanks

**Review by BrittanyIRL — APPROVED** [review]
Beautiful!

**Pull request merged**